### PR TITLE
Updates hub description

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About the Pull Request

Changes hub description to include more config options including:
- Description set from configuration file.
- Discord and Forums links using the config options instead of hard-coded links.
- Less useless features like "vote" and playercount (byond hub already does this for us).
- All of the features will now appear on new lines.

## Why It's Good For The Game

More *useful* information on the hub.

## Did you test it?

Test-merged. It works fine.
image

## Changelog

Not a gameplay change.

### CODE REVIEW

File: config/example/config.txt [hunk 1]
```diff
@@ -1,6 +1,9 @@
 ## Server name: This appears at the top of the screen in-game. In this case it will read "tgstation: station_name" where station_name is the randomly generated name of the station for the round. Remove the # infront of SERVERNAME and replace 'tgstation' with the name of your choice
 # SERVERNAME spacestation13
 
+## Server hub description: This appears on the hub, right below the server name.
+# SERVERDESC Purification in progress.
+
```
Comment (Cupax3) [inline config/example/config.txt]: Can you change this to something more neutral, please?

**Review by Cupax3 — CHANGES_REQUESTED** [review]
Code LGTM, default... eh?

File: config/example/config.txt (hunk 1 above)
Comment (EgorDinamit) [reply config/example/config.txt]: > Can you change this to something more neutral, please?

This is an example config, but sure.

File: config/example/config.txt [hunk 2]
```diff
@@ -1,6 +1,9 @@
 ## Server name: This appears at the top of the screen in-game. In this case it will read "tgstation: station_name" where station_name is the randomly generated name of the station for the round. Remove the # infront of SERVERNAME and replace 'tgstation' with the name of your choice
 # SERVERNAME spacestation13
 
+## Server hub description: This appears on the hub, right below the server name.
+# SERVERDESC Default description of your default server.
+
```
Comment (Cupax3) [inline config/example/config.txt:6]: just the right amount of shade

**Review by Cupax3 — APPROVED** [review]
LGTM

**Pull request merged**